LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Call-Em-All
+Copyright (c) 2014-2016 Call-Em-All
```

Comment:
I'm not a lawyer but here is a relevant StackOverflow [post](http://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated).

I'm not sure if copyright dates are too important, but if it is, I suppose it's up to Call-Em-All to decide since they're the copyright owner. :smile: 

@hai-cea Do you have a preference?